cosmwasm/deployment/terra2/README.md

```diff
@@ -0,0 +1,136 @@
+# Terra2 Wormhole Contract Deployment
+
+This readme describes the steps for building, verifying, and deploying Terra2 smart contracts for Wormhole.
+
+**WARNING**: _This process is only Linux host compatible at this time._
+
+## Verify Tilt
+
+Before building Terra contracts, ensure that the specific commit you will be
+building from passes in tilt. This that ensures basic functionality of the
+Terra smart contracts that you are about to build and deploy.
+
+## Build Contracts
+
+The following command can be used to build Terra2 contracts via Docker.
+
+Build Target Options: [`mainnet`|`testnet`|`devnet`|
+
+These network names correspond to the naming convention used by wormhole
+elsewhere. This means that `mainnet` corresponds to Terra `mainnet`,
+`testnet` corresponds to Terra `testnet`, and `devnet` is `localterra`.
+
+```console
+wormhole/terra2 $ make artifacts
```

Comment:
Let's remove `wormhole/terra2` from these references just for thoroughness?
`make` commands mentioned in here should all just work in this directory (`cosmwasm/deployment/terra2`)

Reply:
sure